app/default_overrides.go

```diff
@@ -263,9 +267,9 @@ func DefaultConsensusConfig() *tmcfg.Config {
 
 	cfg.Mempool.TTLNumBlocks = 12
 	cfg.Mempool.TTLDuration = 75 * time.Second
-	cfg.Mempool.MaxTxBytes = 7_897_088
-	cfg.Mempool.MaxTxsBytes = 39_485_440
-	cfg.Mempool.Version = "v1" // prioritized mempool
+	cfg.Mempool.MaxTxBytes = 2 * mebibyte
+	cfg.Mempool.MaxTxsBytes = 80 * mebibyte
+	cfg.Mempool.Version = "v2" // prioritized mempool
```

Comment:
It shouldn't be a breaking change to set the default to 2MiB since that should already be checked by the antihandler, but there's no reason to keep this at 8

the throughput was quadrupled, and the block times were halved, so the mempool's size was increased